features/features.go

```diff
@@ -687,23 +687,25 @@ var (
 						contactPerson("haircommander").
 						productScope(kubernetes).
 						enhancementPR("https://github.com/kubernetes/enhancements/issues/127").
-						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade, configv1.Default).
 						mustRegister()
 
+	// Note: this feature is perma-alpha, but it is safe and desireable to enable.
```

Comment:
Any links to back up this claim?

Reply:
best I have is https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/127-user-namespaces#pod-security-standards-pss-integration the blurb

```
The Pod Security will relax in a controlled way for pods which enable user namespaces. This behavior can be controlled by an API Server Feature Gate, which allows an early opt-in for end users. The overall burden to ensure that all nodes will honor user namespaces is on the cluster admin, though.
```
the last sentence is what we're able to do now that we have a safe version skew from kube 1.30

Reply:
added a link

Reply:
Do you know if there is a plan to clean this up at some point in the future?